CRM/CiviGeometry/DAO/GeometryEntity.php

```diff
@@ -161,7 +180,7 @@ public function getLog() {
    * @return array
    */
   public static function &import($prefix = FALSE) {
-    $r = CRM_Core_DAO_AllCoreTables::getImports(__CLASS__, 'etry_address_geometry', $prefix, []);
+    $r = CRM_Core_DAO_AllCoreTables::getImports(__CLASS__, 'etry_geometry_entity', $prefix, []);
```

Comment:
This looks odd

Reply:
I'm not sure why but i assume it must be something done with `civicrm` but its just autogenerated